Multiplexor.cpp

```diff
@@ -38,7 +38,6 @@ void Multiplexor::tcaselect(const uint8_t number) {
 
   if (number <= 7) {
   Wire.beginTransmission(TCAADDR);
-  Wire.write(number << 7);
```

Comment:
not what I meant, Marlon. 
What I mean is that you can delete the first two lines if(number > 7) return

Reply:
The line Wire.write(number << 7) is necessary.

Reply:
Also, please tell my why you can delete the first two lines on a personal message.